superset/utils/csv.py

```diff
@@ -49,10 +49,6 @@ def escape_value(value: str) -> str:
     is_negative_number = negative_number_re.match(value) is not None
 
     if needs_escaping and not is_negative_number:
-        # Escape pipe to be extra safe as this
```

Comment:
This is now handled via the `escapechar` character in the `pandas.DataFrame.to_csv` method.